data/successors/successor weapons.txt

```diff
@@ -0,0 +1,389 @@
+		"shield damage" -94
+		# 50 (parent) - 3*48 (child)
+		# close: 50 shield damage
+		# far: 144 shield damage
+		"hull damage" 75
+		# 135 (parent) - 3*20 (child)
+		# close: 135 hull damage
+		# far: 60 hull damage
+		"hit force" 120
+		"cluster"
+		"submunition" "Bimodal Switch" 3
+	description `Precise and deadly, the "Ojet" Bimodal Coilgun has been the High Houses' armament of choice since long before humans ever took to the stars. It was designed quite simply as a perfect weapon; it is said that by the time the empire of the Predecessors fell, they had seen no need to install any other weapon on their ships for over a thousand years.`
+	description `	The "Ojet" fires hyperdense packets of charged particles that disperse as they travel: up close the concentrated bolt pierces armor, while at a distance the ionic spray destabilizes shielding. Together, a ship equipped with these weapons can adjust its damage profile just by changing its engagement range, allowing it to be highly effective against any foe.`
+
+outfit "Bimodal Coilgun Turret"
+	category "Turrets"
+	series "Turrets: Successors"
+	index 10
+	cost 2950000
+	thumbnail "outfit/bimodal coilgun turret"
+	licenses
+		"High Houses"
+	"mass" 55
+	"outfit space" -55
+	"weapon capacity" -55
+	"turret mounts" -1
+	weapon
+		"hardpoint sprite" "hardpoint/bimodal turret"
+		"hardpoint offset" 12
+		sprite "projectile/bimodal"
+			"frame rate" 5
+			"start frame" 0
+			"no repeat"
+		sound "bimodal"
+		"hit effect" "bimodal impact"
+		"fire effect" "moonbeam fleck" 4
+		"inaccuracy" 1
+		"velocity" 30
+		"lifetime" 15
+		"range override" 900
+		"turret turn" 3.5
+		"burst count" 24
+		"burst reload" 3
+		"reload" 15
+		"firing energy" 80
+		"firing heat" 120
+		"firing force" 25
+		"shield damage" -94
+		"hull damage" 75
+		"hit force" 120
+		"cluster"
+		"submunition" "Bimodal Switch" 3
+	description `Successor warships have traditionally relied on fixed weaponry to deal with their foes, but the High Houses do produce a turreted platform for the "Ojet" Coilgun so that their larger and slower vessels can maintain their combat effectiveness against more maneuverable targets. It mounts two of the weapons side by side, rigged to fire double the amount of shots per burst to keep them in sync with the fixed hardpoints.`
+
+effect "bimodal impact"
+	sprite "effect/bimodal impact"
+		"frame rate" 30
+		"no repeat"
+	lifetime 8
+	"velocity scale" 0.2
+
+outfit "Bimodal Switch"
+	weapon
+		sprite "projectile/bimodal switch"
+			"random start frame"
+			"frame rate" 20
+		"hit effect" "bimodal switch impact"
+		"inaccuracy" 10
+		"velocity" 30
+		"lifetime" 6
+		"random lifetime" 3
+		"shield damage" 48
+		"hull damage" 20
+
+effect "bimodal switch impact"
+	sprite "effect/bimodal switch impact"
+		"frame rate" 12
+		"no repeat"
+	lifetime 30
+	"velocity scale" 0.02
+
+outfit "Drag Cannon"
```

Comment:
The Drag Cannon has very high DPS for its size, applies a small amount of slowing damage, and benefits from a clustered burst firing mode that allows it to output large amounts of damage in a short space of time - all of which would make it a very strong weapon. On the other hand, the range is very low, and with 20 degrees of inaccuracy the effective damage to all but the largest targets at anything beyond point-blank range is much more reasonable.